src/nvim/window.c

```diff
@@ -3834,17 +3834,16 @@ int tabpage_index(tabpage_T *ftp)
   return i;
 }
 
-/*
- * Prepare for leaving the current tab page.
- * When autocommands change "curtab" we don't leave the tab page and return
- * FAIL.
- * Careful: When OK is returned need to get a new tab page very very soon!
- */
-static int 
-leave_tabpage (
-    buf_T *new_curbuf,        /* what is going to be the new curbuf,
-                                          NULL if unknown */
-    int trigger_leave_autocmds
+/// Prepare for leaving the current tab page.
+/// When autocommands change "curtab" we don't leave the tab page and return
+/// FAIL.
+/// Careful: When OK is returned need to get a new tab page very very soon!
+///
+/// @param  new_curbuf   what is going to be the new curbuf, NULL if unknown
+static int
+leave_tabpage(
+    buf_T *new_curbuf,
+    bool trigger_leave_autocmds
```

Comment:
Do you think you'd be able to add documentation for the `trigger_leave_autocmds` parameter ?